spec/Section 6 -- Execution.md

```diff
@@ -26,13 +26,21 @@ request is determined by the result of executing this operation according to the
 "Executing Operations” section below.
 
 ExecuteRequest(schema, document, operationName, variableValues, initialValue):
+Note: the execution assumes implementing language supports coroutines.
+Alternatively, the socket can provide a write buffer pointer to allow
+{ExecuteRequest()} to directly write payloads into the buffer.
 
 - Let {operation} be the result of {GetOperation(document, operationName)}.
 - Let {coercedVariableValues} be the result of {CoerceVariableValues(schema,
   operation, variableValues)}.
 - If {operation} is a query operation:
-  - Return {ExecuteQuery(operation, schema, coercedVariableValues,
-    initialValue)}.
+  - Let {executionResult} be the result of calling {ExecuteQuery(operation,
```

Comment:
@robrichard Is it possible this edit is not necessary considering the changes to ExecuteQuery below? Or, is there a different reason why queries are treated differently than mutations in this section?